torch/fx/experimental/proxy_tensor.py

```diff
@@ -1789,7 +1789,11 @@ def maybe_handle_decomp(
         args: Tuple[object, ...],
         kwargs: Dict[str, object]
 ) -> object:
+    from torch._inductor.bisect_helper import BisectionManager
     if op in CURRENT_DECOMPOSITION_TABLE:
+        if BisectionManager.disable_subsystem("aot_eager_decomp_partition", "decomposition", lambda: repr(op)):
```

Comment:
But... this is different from how aot_eager does it? Specifically, doesn't aot _eager just not set CURRENT_DECOMPOSITION_TABLE?

Reply:
That's fine - we iterate over aot_eager specifically as a backend.